src/csvs_to_pdfs.py

```diff
@@ -0,0 +1,115 @@
+"""
+Sweeps all the apple pick bagfiles and saves all the plots from a certain variable in a pdf,
+and for a certain label (e.g. Succesful or Failure)
+Created by: velasale@oregonstate.edu
+
+References:
+https://stackoverflow.com/questions/38938454/python-saving-multiple-subplot-figures-to-pdf
+"""
+
+# ... System related packages
+import os
+import time
+from os.path import exists
+from tqdm import tqdm
+# ... File related packages
+import pandas as pd
+import csv
+# import bagpy
+# from bagpy import bagreader
+# ... Math related packages
+import math
+import numpy as np
+from scipy.ndimage.filters import gaussian_filter, median_filter
+# ... Plot related packages
+import matplotlib.pyplot as plt
+from matplotlib.backends.backend_pdf import PdfPages
+# import seaborn as sns
+
+# sns.set()  # Setting seaborn as default style even if use only matplotlib
+plt.close('all')
+
+# Step 1 - Open csv
+
+main = 'C:/Users/15416/Box/Learning to pick fruit/Apple Pick Data/RAL22 Paper/'
+datasets = ['3_proxy_winter22_x1', '5_real_fall21_x1']
```

Comment:
This could be made command line specifiable to be more user-friendly. That way you won't have to open and edit the code to process new datasets.